lib/qldb.base.model.js

```diff
@@ -11,14 +11,11 @@ class Ledger {
 
         this.options = options;
         if (this.options.timestamps == true) {
-            let date = new Date().toISOString();
             this.model['createdAt'] = {
                 type: DataTypes.TIMESTAMP,
-                value: date
             };
             this.model['updatedAt'] = {
                 type: DataTypes.TIMESTAMP,
-                value: date
```

Comment:
Why not set in the constructor?

Reply:
if we set it in the constructor, createdAt/updatedAt values will be same for all the documents that are going to be inserted/updated in future at different dates(as constructor is an initializing object). so assigned a value in the model instead of constructor for specific operation(create/update). correct me if I'm wrong.

Reply:
I don't believe so, when you create or update a new document it will instantiate a new one and thus the values are set again to "now"The constructor is called on each new model that is created, not only once at startup.

Reply:
can you check that once? I'm still seeing the old updatedAt value even after updating the document after some time. I heard that you're using lambda for your application so that might be the reason you're getting new instance of it and values reflecting correctly on every call. but for my local, it is still showing older date(now) instance when I actually started the server.

Reply:
Correct, I'm running this on lambda and locally I use serverless-offline. Could be the reason for it as why I'm not getting your results. But then again, even when I'm building a regular MySQL connector on a local machine I get a new instance on every call. Are you sure that you want to keep you connection live for the duration of forever? Aren;t you afraid of memory leaks etc messing your server up? I always close my connections when I'm done with the call.

Reply:
we're already closing the session after the tx completed. in my case, the server should be always up and running. so it doesn't seem to be updating document timestamps correctly